extensions/amp-accordion/1.0/accordion.js

```diff
@@ -152,11 +229,12 @@ export function AccordionSection({
   animate: defaultAnimate = false,
   headerClassName = '',
   contentClassName = '',
+  id,
   header,
   children,
   ...rest
 }) {
-  const [id] = useState(generateSectionId);
+  const [internalId] = useState(id || generateSectionId);
```

Comment:
This is clever, but it's better to do

```
const [genId] = useState(generateSectionId);
const internalId = id || genId;
```

This is better b/c the change of `id` is immediately reflected.

Reply:
Thanks for the suggestion, updated!